NEWS

```diff
@@ -10,6 +10,8 @@ DON'T FORGET TO BUMP THE -version-info PRE-RELEASE IF NECESSARY!
 * Issue #23: Fix double-free triggered by atf_map_insert in low memory
   scenarios, caused by an overlook in the atf_list code.
 
+* PR #32: Add a subroutine to evaluate inequality of arguments.
```

Comment:
Please be specific. People reading the release notes should be able to tell when new APIs were exactly introduced so that they can decide whether to rely on a newer ATF version or not.

I'd say:

* Added the atf_check_not_equal function to atf-sh to check for unequal values.

or something like this.